src/web/jcc/client/components/SettingsPanel.tsx

```diff
@@ -54,14 +55,19 @@ enum AccordionTabs {
 }
 
 /**
- * Defines Layout and Functionality of the Settings Panel (Map Settings)
+ * Defines Layout and Functionality of the SettingsPanel
  *
  * @param {Props} Props passed from CommandControl
  * @returns {string} Web content of Settings Panel
  */
 
 export function SettingsPanel(props: Props) {
     const [openAccordionTabs, setOpenAccordionTabs] = useState([])
+    const [accordionTheme, setAccordionTheme] = useState(createTheme({
+        transitions: {
+            create: () => 'none',
+        }
+    }))
 
     const handleClusterToggleClick = () => {
```

Comment:
Can we add a function comment

Reply:
I'm not sure we need a function comment here. We are calling the createTheme function from MUI.

Reply:
Since it is outside the scope of the code I modified would it be ok to hold off on adding the function comment here?